src/main/java/pikachu/command/AddCommand.java

```diff
@@ -0,0 +1,60 @@
+package pikachu.command;
+import java.time.LocalDate;
+
+import pikachu.PikachuException;
+import pikachu.Storage;
+import pikachu.TaskList;
+import pikachu.Ui;
+import pikachu.task.Deadline;
+import pikachu.task.Event;
+import pikachu.task.Todo;
+
+public class AddCommand extends Command {
+    String input;
+
+    public AddCommand(String fullCommand) {
+        input = fullCommand;
+    }
+
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws PikachuException {
+        String tempStr = "";
+        if (input.startsWith("deadline ") && input.contains(" /by ")) {
+            String temp1 = input.split(" ",2)[1];
+            String[] temp2 = temp1.split(" /by ",2);
+            try {
+                LocalDate date = LocalDate.parse(temp2[1]);
+                Deadline newDDL = new Deadline(temp2[0],date);
+                tasks.taskList.add(newDDL);
+                tempStr = "Pikapi(added): " + newDDL + '\n';
+                tempStr += "Pikaaaaa: " + tasks.taskList.size() + (tasks.taskList.size() > 1 ? " tasks" : " task");
```

Comment:
Perhaps for long line of codes, can break into 2 sentence!

Reply:
fixed! Thank you!